commit bafa8f30b6f83d5f02bb65bb47b2943a0cb3c290
Author: Cory Bloor <Cordell.Bloor@amd.com>
Date:   Wed Jan 25 14:52:35 2023 -0700

    Fix use of fmt 9.0 and later in logging test (#515)
    
    The implicitly defined formatter that rocsolver was using for
    std::filesystem::path has been removed from fmt 9.0 and later.
    
    This change doesn't actually fix compatibility with the official
    fmt 9.0 or 9.1 releases, because those releases are still
    incompatible with HIP. However, the change is sufficient for using
    rocsolver with fmt@9 from Spack (because the fix has been backported).

diff --git a/clients/gtest/logging_gtest.cpp b/clients/gtest/logging_gtest.cpp
index e4f594a..da7ae04 100644
--- a/clients/gtest/logging_gtest.cpp
+++ b/clients/gtest/logging_gtest.cpp
@@ -1,5 +1,5 @@
 /* ************************************************************************
- * Copyright (c) 2022 Advanced Micro Devices, Inc.
+ * Copyright (c) 2022-2023 Advanced Micro Devices, Inc.
  * ************************************************************************ */
 
 #include <cstdlib>
@@ -49,7 +49,7 @@ protected:
         {
             if(HasFailure() && std::getenv("ROCSOLVER_TEST_DEBUG"))
                 fmt::print(stderr, "ROCSOLVER_TEST_DEBUG is set so {} was not removed.\n",
-                           log_filepath);
+                           log_filepath.string());
             else
                 EXPECT_TRUE(fs::remove(log_filepath));
         }
